<launch>
  <group ns = "test_robot_1">
    <node pkg="mdis_state_machine" type="team_scheduler" name="team_scheduler" args="5 dummy_parent dummy_child" output="screen"/>
  </group>
</launch>